packages/grant-explorer/src/features/api/rounds.ts

```diff
@@ -65,7 +66,7 @@ async function fetchRoundsByTimestamp(
         round.roundMetaPtr.pointer
       );
       round.roundMetadata = roundMetadata;
-      round.chainId = chainId;
+      round.chainId = chainId.toFixed(0);
```

Comment:
what's the scenario where we need `toFixed` here?

Reply:
<img width="894" alt="image" src="https://github.com/gitcoinco/grants-stack/assets/10381200/571d1042-8172-48ed-95e0-d790feead9a3">